Frontend/android-native/app/src/main/java/in/juspay/mobility/utils/RideRequestActivity.java

```diff
@@ -258,7 +258,9 @@ public void run() {
                         rideRequestBundle.getInt("rideRequestPopupDelayDuration"),
                         negotiationUnit,
                         rideRequestBundle.getInt("customerTip"),
-                        rideRequestBundle.getString("specialLocationTag"));
+                        rideRequestBundle.getString("specialLocationTag"),
```

Comment:
Add the pincode visibility conditions in this Activity's `onBindViewHolder` as well